README.md

```diff
@@ -117,6 +129,43 @@ healthy
   -p 25565:25577
   ```
 
+## Interacting with the server
+
+[RCON](http://wiki.vg/RCON) is enabled by default, so you can `exec` into the container to
+access the Minecraft server console:
```

Comment:
copy-paste left over from minecraft server image?
```suggestion
access the Bungeecord server:
```

Reply:
Yes and visibly i have forgot some change